frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/MenuLayout.java

```diff
@@ -136,7 +146,17 @@ public void buttonClick(Button.ClickEvent event) {
                 authService.logout(RequestHolder.getRequest());
                 authCtx.clear();
                 refreshUserBar();
-                navigator.navigateTo(Login.class);
+                String logout_url = null;
+                try {
+                    logout_url = appConfig.getString(ConfigProperty.LOGOUT_URL);
```

Comment:
The ConfigPropety.LOGOUT_URL should be renamed to ConfigPropety.SSO_LOGOUT_URL, so that it is clear that for standalone use it is not needed